README.md

```diff
@@ -191,6 +192,41 @@ EXAMPLES
 
 _See code: [src/commands/create.ts](https://github.com/superfaceai/cli/tree/main/src/commands/create.ts)_
 
+## `superface compile`
```

Comment:
Nice! But :/ 
CLI uses oclif that have custom cli to generate command descriptions so we can do this automatically.

Reply:
Oh!

So should I re-generate this, or is this fine as it is? :)

Reply:
Compile command is hidden for now so generation won't add it's description to readme. But after I make compile visible your changes in code itself will be propagated to readme. So I think it is fine.